addons/l10n_es_edi_tbai/models/l10n_es_edi_tbai_document.py

```diff
@@ -218,9 +218,11 @@ def _send_request_to_agency(*args, **kwargs):
                 error_code = response_headers['eus-bizkaia-n3-codigo-respuesta']
                 error_msg = response_headers['eus-bizkaia-n3-mensaje-respuesta']
                 errors.append(error_code + ": " + error_msg)
-            if errors:
-                return False, errors
-            return self._process_post_response_xml_bi(env, response_xml)
+            success = False
+            if response_xml is not None:
```

Comment:
We should put this logic (the if condition) inside `_process_post_response_xml_bi`. It makes the behavior more coherent if we need to use it somewhere else.

Here we only have something like this:

```suggestion
     success, errors_add = self._process_post_response_xml_bi(env, response_xml)
     errors += errors_add
     return success, errors
```